nexus/src/app/sagas/snapshot_create.rs

```diff
@@ -145,9 +158,38 @@ lazy_static! {
         ssc_create_snapshot_record,
         ssc_create_snapshot_record_undo,
     );
-    static ref SEND_SNAPSHOT_REQUEST: NexusAction = new_action_noop_undo(
-        "snapshot-create.send-snapshot-request",
-        ssc_send_snapshot_request,
+    static ref SEND_SNAPSHOT_REQUEST_TO_SLED_AGENT: NexusAction =
+        new_action_noop_undo(
+            "snapshot-create.send-snapshot-request-to-sled-agent",
+            ssc_send_snapshot_request_to_sled_agent,
+        );
```

Comment:
If the saga fails after this point, will this leave behind a snapshot?

Reply:
Between work from @smklein and new undo steps added as part of the current rev, there should be no more
left behind snapshots.